player/loadfile.c

```diff
@@ -300,19 +290,21 @@ static void print_stream(struct MPContext *mpctx, struct track *t)
         }
     } else if (t->type == STREAM_AUDIO) {
         if (s && s->codec->channels.num)
-            APPEND(b, " %d ch", s->codec->channels.num);
-        if (s && s->codec->samplerate) {
-            char *samplerate = mp_format_double(NULL, s->codec->samplerate / 1000.0,
-                                                4, false, false, true);
-            APPEND(b, " %s kHz", samplerate);
-            talloc_free(samplerate);
-        }
+            APPEND(b, " %dch", s->codec->channels.num);
+        if (s && s->codec->samplerate)
+            APPEND(b, " %d Hz", s->codec->samplerate);
     }
-    APPEND(b, ")");
     if (s && s->hls_bitrate > 0)
-        APPEND(b, " (%d kbps)", (s->hls_bitrate + 500) / 1000);
+        APPEND(b, "  %d kbps", (s->hls_bitrate + 500) / 1000);
     if (t->is_external)
-        APPEND(b, " (external)");
+        APPEND(b, " external");
+    if (t->default_track)
+        APPEND(b, " default");
+    if (t->forced_track)
+        APPEND(b, " forced");
+    if (t->attached_picture)
+        APPEND(b, " picture");
+    APPEND(b, ")");
```

Comment:
Instead:
`(ac3 6ch 48000 Hz default external)`
I would prefer it separated, to avoid too much clutter:
`(ac3 6ch 48000 Hz) [default external]`
Things in `()` varies by track, but flags are common, so they would be more consistent together in `[]` imho